woocommerce/woocommerce-ios
# Update dark-mode icons in SuverySubmittedVC

# Why

Prior to this PR the `SurveySubmitted` screen didn't have proper dark mode images and icons. This PR adds just that.

# Screenshots

Light | Dark
--- | ---
<img width="419" alt="light" src="https://user-images.githubusercontent.com/562080/89316713-14995480-d642-11ea-82ce-33a5b37a2f9b.png"> | <img width="425" alt="dark" src="https://user-images.githubusercontent.com/562080/89316723-16fbae80-d642-11ea-9f8c-017afc9ac1ff.png">

# Testing Steps
- Add the following code on `InAppFeedbackCardViewController` in the `configureDidNotLikeButton` method
```swift
didNotLikeButton.on(.touchUpInside) { [weak self] _  in
    let vc = SurveySubmittedViewController()
    let nav = WooNavigationController(rootViewController: vc)
    self?.present(nav, animated: true)
}
```
- Launch the app in dark mode
- Tap on the "Could Be Better" button
- See the icons properly rendered in dark mode

Update release notes:
